drivers/SmartThings/zigbee-switch/fingerprints.yml

```diff
@@ -1983,6 +2000,14 @@ zigbeeGeneric:
         - 0x0008 #Level
         - 0x0300 #Color Control
     deviceProfileName: rgbw-bulb
+  - id: "genericTWBulb"
+    deviceLabel: Zigbee Light
+    clusters:
+      server:
+        - 0x0006 #OnOff
+        - 0x0008 #Level
+        - 0x0300 #Color Control
+    deviceProfileName: color-temp-bulb
```

Comment:
this is a duplicate of the fingerprint above 

please remove

Reply:
fixed.